src/tokens/solana.tokenlist.json

```diff
@@ -21046,7 +21046,22 @@
         "website": ""        
       }
     }
+  ,{
+  "chainId": 101,
+  "address": "F7mgxaYF1gg1hBtaVzENSG6ey3pn6J1mXhBRmHxDzBNg ",
+  "symbol": "SINU ",
+  "name": "Samo INU ",
```

Comment:
This space is causing issues in address parsing, needs to be addressed. Spaces also in symbol/name as well.

Reply:
Hello ebramanti,

please how can this be corrected?

Thans

Reply:
Remove trailing spaces at the end of all of these lines